src/resources/glossary.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Glossary
+description: A glossary reference for terminology used across dart.dev
+sitemap: false
+---
+
+The following are definitions of terminology used across the Dart documentation.
+
+## Irrefutable patterns
+
+_Irrefutable patterns_ are patterns that always match. 
+Irrefutable patterns appear in _irrefutable contexts_: 
```

Comment:
To me, "appear" implies "can only appear", but irrefutable patterns can also appear in a matching context. For example
```
void f((int, int) r) {
  switch (r) {
    case (0, var y):
      // `y` is bound to the second field if the first field is zero (0).
  }
}
```

Consider "Irrefutable patterns appear" --> something like "Only irrefutable patterns can appear"